build_guest

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -fvx 
+
+wget https://cdn.kernel.org/pub/linux/kernel/v6.x/linux-6.12.5.tar.xz
+mkdir -p linux
+cd linux && xzcat ../linux-6.12.5.tar.xz | tar xf -
+cd linux-6.12.5
+cp ../../cpu.config .config
+make -j $(nproc) Image
```

Comment:
building in the same directory as kernel source is evil and should be avoided.  Use a .build or alternate build directory and use O=$(LINUX_BUILD)

Reply:
Yeah this is wrong and I will fix it.